styles/style_demo.php

```diff
@@ -9,16 +9,26 @@
 
 <h1>Style Demos</h1>
 
+<div class='quick-links'>
```

Comment:
This is great, although I was thinking more in the "here's how to do quick links" sense. I think all you need to do is add an `<h2>Quick Links</h2>` above this line.

Reply:
Added and moved below introductory text explaining the page.